examples-tests/pnpm-gatsby/test.t

```diff
@@ -0,0 +1,29 @@
+  $ . ${TESTDIR}/../setup.sh with-gatsby pnpm
+# run twice and make sure it works
+  $ pnpm run build lint -- --output-logs=none
+  
+  \> with-gatsby@0.0.0 build (.*)/test.t (re)
+  \> turbo build "lint" "--output-logs=none" (re)
+  
+  • Packages in scope: docs, eslint-config-custom, tsconfig, ui, web (esc)
+  • Running build, lint in 5 packages (esc)
+  • Remote caching disabled (esc)
+  
+   Tasks:    5 successful, 5 total
+  Cached:    0 cached, 5 total
+    Time:\s*[\.0-9]+m?s  (re)
+  
+  $ pnpm run build lint -- --output-logs=none
+  
+  \> with-gatsby@0.0.0 build (.*)/test.t (re)
+  \> turbo build "lint" "--output-logs=none" (re)
+  
+  • Packages in scope: docs, eslint-config-custom, tsconfig, ui, web (esc)
+  • Running build, lint in 5 packages (esc)
+  • Remote caching disabled (esc)
+  
+   Tasks:    5 successful, 5 total
+  Cached:    3 cached, 5 total
```

Comment:
Not going to try to fix this here, but interesting that `with-gatsby` is not getting Full Turbo on the second run